NEWS.md

```diff
@@ -17,13 +17,13 @@ wb_dims(x = mtcars, cols = "non-existent-col")
 # Now errors
 ```
 
+* `wb_set_col_widths()` is more strict about its arguments. If you provide `cols`, `widths`, or `hidden` don't have appropriate length, it will throw an error.
+
 
 ***************************************************************************
 
 # openxlsx2 1.5
 
-(This was updated post release.)
-
```

Comment:
Unrelated, buts this should stay, because it significantly differs from the CRAN tar file and the reader should not doubt the tar file.
I don’t care for minor changes like typos or forgotten or fixed links, but here I converted a breaking change to a new feature.